drivers/periph_common/hwcrypto.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2016-2018 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers
+ * @{
+ *
+ * @file
+ * @brief       Shared hardware crypto code
+ *
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * @}
+ */
+
+#include "periph/hwcrypto.h"
+
+#ifndef PERIPH_HWCRYPTO_PROVIDES_CIPHER_SUPPORTED
+/**
+ * @brief Bitmap of supported ciphers.
+ */
+static const uint32_t cipher_bitmap = (
+#ifdef HAVE_HWCRYPTO_AES128
```

Comment:
I propose using `|` instead of `+` for chaining these. The error case (duplicate entry or meaning) is less ugly...